src/Dashboard.js

```diff
@@ -22,66 +24,67 @@ export default class Dashboard extends React.Component {
       }, {});
     });
     this.columns = [
-      {
-        Header: "",
-        accessor: "id",
-        className: "dv-Dashboard__icon",
-        headerClassName: "not-sortable",
-        sortable: false,
-        minWidth: 50,
-        Cell: ({ value }) => (
-          <button
-            onClick={e => this.handleOpenModal(value)}
-            className="dv-Dashboard__button"
-          >
-            <ModalIcon />
-          </button>
-        )
-      },
       ...Object.keys(this.props.tableData[0])
-        .filter(d => d !== "id" && d !== "Link")
+        .filter(d => d !== "id")
         .map(d =>
           d === "Project"
-            ? {
-                Header: d,
-                accessor: d,
-                minWidth: 150,
-                Cell: row => {
-                  const link = row.original["Link"];
-                  if (link) {
-                    return (
-                      <a target="_blank" rel="noopener noreferrer" href={link}>
-                        {row.value}
-                      </a>
-                    );
-                  } else {
-                    return row.value;
-                  }
-                }
-              }
-            : d === "Description"
-            ? {
-                Header: d,
-                accessor: d,
-                minWidth: 245
-              }
-            : {
-                Header: d,
-                accessor: d,
-                minWidth: 150
+          ? {
+            Header: d,
+            accessor: d,
+            minWidth: 180,
+            Cell: row => {
+              const id = row.original["id"];
+              if (id) {
+                return (
+                  <div>
+                    <ReactMarkdown
+                      source={row.value}
+                      className="dv-ReactMarkdown"
+                      linkTarget="_blank"
+                    />
+                    <button
+                      onClick={e => this.handleOpenModal(id)}
+                      className="dv-Dashboard__button dv-Dashboard__button--open-modal"
+                    >
+                      Read more
+                    </button>
+                  </div>
+                );
+              } else {
+                return row.value;
               }
+            }
+          }
+          : {
+            Header: d,
+            accessor: d,
+            minWidth: d === "Description" ? 245 : 150,
+            Cell: row => (
+              <ReactMarkdown
+                source={row.value}
+                className="dv-ReactMarkdown"
+                linkTarget="_blank"
+              />
+            )
+          }
         )
     ];
     this.dataMap = group(this.props.popupData, d => d.id);
     this.handleOpenModal = this.handleOpenModal.bind(this);
     this.handleCloseModal = this.handleCloseModal.bind(this);
+    this.handleSidebarExpand = this.handleSidebarExpand.bind(this);
```

Comment:
I'm not convinced I named this very well, open to suggestions (not that this matters very much)